addons/im_livechat/static/src/bugfix/bugfix.js

```diff
@@ -7,4 +7,28 @@
 odoo.define('im_livechat/static/src/bugfix/bugfix.js', function (require) {
 'use strict';
 
+const {
+    registerInstancePatchModel,
+} = require('mail/static/src/model/model_core.js');
+
+registerInstancePatchModel('mail.chat_window', 'im_livechat/static/src/bugfix/bugfix.js', {
```

Comment:
Usually in bugfix file we keep the name of the path as it would be in the final version to have an easier time just move it to its new file, but this is fine too. Just need to keep in mind to rename this when moving to own file in master